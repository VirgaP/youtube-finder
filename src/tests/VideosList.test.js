import React from 'react';
import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import VideoList from '../components/VideosList';

configure({adapter: new Adapter()});

describe(VideoList, () => {
    const component = shallow(<VideoList data={[]}/>);
    it('should exist', () => {
        const component = renderer.create(<VideoList data={[]}/>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    }) 
});