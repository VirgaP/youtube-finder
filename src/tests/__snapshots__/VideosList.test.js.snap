// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideosList should exist 1`] = `
<div
  className="video-list-container"
  id="videosList"
/>
`;
